org.kde.spectacle.json

```
@@ -0,0 +1,84 @@
+{
+  "id": "org.kde.spectacle",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-22.08",
+  "sdk": "org.kde.Sdk",
+  "command": "spectacle",
+  "rename-icon": "spectacle",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
```

Doesn't work in wayland, see comments https://github.com/flathub/flathub/pull/3181#issuecomment-1763233343 and https://github.com/flathub/flathub/pull/3181#issuecomment-1763237059
```suggestion
    "--socket=x11",
```

---

Actually this was removed in https://github.com/flathub/flathub/compare/4f1391f8fd6b3000358f4a38038177e43967eb31..a8a5b93b5e6bb1a736249fdf2a2aef99e7d1f3c6 why is it back?